src/styles/utils/variable.scss

```
@@ -0,0 +1,5 @@
+$color-main: #060b35;
+$color-secondary: #616070;
+$color-button-blue: #00acdc;
+$color-button-active: #fff;
+$width-card: 200px;
```

You have variables but you don't use them.

Be careful and use more universal naming.
```suggestion
$color-main: #060b35;
$color-secondary: #616070;
$color-button: #00acdc;
$color-button-active: #fff;
$width-card: 200px;
```